cmd/aro/operator.go

```diff
@@ -237,12 +238,17 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 			arocli, operatorcli)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller %s: %v", ingress.ControllerName, err)
 		}
+		if err = (checker.NewReconciler(
+			log.WithField("controller", checker.ControllerName),
+			arocli, kubernetescli, maocli, operatorcli, configcli, role)).SetupWithManager(mgr); err != nil {
+			return fmt.Errorf("unable to create controller %s: %v", checker.ControllerName, err)
+		}
```

Comment:
Only internet checker needs to be run on workers. Everything else - runs on master. So moving the whole thing into master.